code/game/objects/structures/xeno.dm

```diff
@@ -745,11 +746,14 @@ TUNNEL
 		var/area/A = get_area(src)
 		if(A)
 			to_chat(creator, "<span class='xenoannounce'>You sense your acid well at [A.name] has been destroyed!</span>")
-	var/datum/effect_system/smoke_spread/xeno/acid/A = new(get_turf(src))
-	A.set_up(clamp(charges,0,2),src)
-	A.start()
+	addtimer(CALLBACK(src,.proc/release_smoke), 1 SECONDS)
```

Comment:
this wont work because the timer will be deleted in the parent destroy

Reply:
I just tested it, and it seems to work.